### PR TITLE
refactor: update models and tests

### LINKED ISSUE #57: Update certain numeric values to strings
After https://github.com/ArkEcosystem/core/pull/2739 some models/structs will need adjustments.

### PR DESCRIPTION
**Pull request by dated**
## Summary

Updates the datatypes and tests to reflect the current development state, closes #57.

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

**Comment by dated** [comment]
Did you get a chance to take a look at this @faustbrian? This PR is sort of a requirement for #74.

File: client/transactions_responses.go [hunk 1]
```diff
@@ -12,19 +12,21 @@ import (
...
+	Fee             uint64            `json:"fee,omitempty,string"`
+	Sender          string            `json:"sender,omitempty"`
+	SenderPublicKey string            `json:"senderPublicKey,omitempty"`
+	Recipient       string            `json:"recipient,omitempty"`
+	Signature       string            `json:"signature,omitempty"`
+	VendorField     string            `json:"vendorField,omitempty"`
+	Asset           *TransactionAsset `json:"asset,omitempty"`
+	Confirmations   uint32            `json:"confirmations,omitempty"`
+	Timestamp       Timestamp         `json:"timestamp,omitempty"`
+	Nonce           string            `json:"nonce,omitempty"`
```
Comment (dated) [inline client/transactions_responses.go]: This should be changed to `uint32` / `uint64` :thinking:

**Comment by faustbrian** [comment]
@dated can you resubmit this against `master`

**Comment by dated** [comment]
Done, #78.

**Pull request closed without merging**